### ISSUE TITLE
`operationNameGenerator` enum not resolved from .refitter file

### ISSUE THREAD

**Issue by Xeevis** [christianhelle/refitter#277]
**Describe the bug**
When using the CLI `refitter --settings-file` as specified by documentation
```json
{
    "openApiPath": "swagger.yaml",
    "operationNameGenerator": "Default"
}
```
it throws
> Error: The JSON value could not be converted to Refitter.Core.OperationNameGeneratorTypes. Path: $.operationNameGenerator | LineNumber: 2 | BytePositionInLine: 39.

Integer works fine
```json
"operationNameGenerator": 0
```

**Additional context**
Version 0.9.0

**Comment by christianhelle** [comment]
@Xeevis Ooops! I missed that detail... one-liner fix on deserializing enums as strings on the way in PR #278 

Sorry for the inconvenience and thanks for taking the time to report this

**Comment by christianhelle** [comment]
@Xeevis fix is released as [v0.9.2](https://www.nuget.org/packages/Refitter/0.9.2)
